### PR TITLE
CI: Add a GHA workflow to track first-time pull request and issue creators

### PR DESCRIPTION
**Pull request by HarshCasper**
## Description

This PR:

- Adds a GHA workflow to track first-time pull requests and issue creators
- Directs issue creators towards Slack workspace and Pro support bot 
- Directs issue creators and pull request creators to community guidelines, code of conduct and develop guide

### CODE REVIEW

**Review by thrau — APPROVED** [review]
looks awesome! looking forward to seeing this in action :-)

File: .github/workflows/welcome-first-time-contributors.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          const prMessage = `Welcome to LocalStack! Thanks for raising your first Pull Request and landing in your contributions. Our team will reach out with any reviews or feedbacks that we have shortly. We recommend joining our [Slack Community](https://join.slack.com/t/localstack-community/shared_invite/zt-10v01mnte-tHiu0CTFdaGHYomID3WNcQ) and share your PR on the **#community** channel to share your contributions with us. Please make sure you are following our [contributing guidelines](https://github.com/localstack/.github/blob/main/CONTRIBUTING.md) and our [Code of Conduct](https://github.com/localstack/.github/blob/main/CODE_OF_CONDUCT.md).`;
+
+          if (!issueMessage && !prMessage) {
+              throw new Error('Action should have either issueMessage or prMessage set');
+          }
+
+          const isIssue = !!context.payload.issue;
+          let isFirstContribution;
+          if (isIssue) {
+              const query = `query($owner:String!, $name:String!, $contributer:String!) {
```
Comment (whummer) [inline .github/workflows/welcome-first-time-contributors.yml:29]: Amazing - GraphQL! ❤️

**Review by whummer — APPROVED** [review]
Looks great @HarshCasper . 👍  Only thing we should briefly discuss is whether we want the public Slack link added automatically..

**Comment by HarshCasper** [comment]
@whummer I have added LocalStack's Contact Page link which folks can reference to join the Slack community.

**Pull request merged**